CHANGES/4650.bugfix

```diff
@@ -0,0 +1 @@
+Modify test_utils.BaseTestServer to allow IPv6 hostnames.
```

Comment:
I think that the user-facing message should be something valuable to them, not the implementation detail:
```suggestion
Implement binding to IPv6 addresses in the pytest server fixture.
```

WDYT?

Reply:
Sounds good!